instancio-core/src/main/java/org/instancio/generator/specs/CoordinateGeneratorSpec.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.instancio.generator.specs;
+
+/**
+ * Spec for generating coordinate values.
+ *
+ * @since 4.3.3
```

Comment:
If you'd like to include this PR in the next release, the next will be 4.4.0. I will skip 4.3.3 as there are quite a few changes since the last release.

Reply:
Updated to 4.4.0 across the different doc sections in current squashed PR commit, next release would be awesome!